lib/pure/asyncfile.nim

```diff
@@ -466,6 +466,38 @@ proc write*(f: AsyncFile, data: string): Future[void] =
       addWrite(f.fd, cb)
   return retFuture
 
+proc len*(f: AsyncFile): int64 =
```

Comment:
ahh, I think we should have ``getFileSize`` and ``setFileSize`` for both. i.e. rename ``len`` to ``getFileSize`` and ``setLen`` to ``setFileSize``.